### DISCUSSION TITLE
The tests in the tests directory are encountering errors when running on pytest.

### DISCUSSION THREAD

**Discussion by Wangzixu0426** [pnnl/neuromancer#202] [Q&A] [answered]
I encountered the following issues when running tests on pytest. Are these issues related to me personally?
Ⅰ.The issue in the test_blocks.py module was resolved by adding the missing imports for KANBlock and StackedMLP.When I imported these two modules, the errors disappeared. If I do not import these two modules in the import section of the tests, will it affect my normal usage? The third question is similar to this one.
Ⅱ.In the tests test_linear_shape, test_mlp_shape, test_res_mlp_shape, test_rnns_shape, etc., a TypeError: unsupported operand type(s) for +: 'Tensor' and 'NoneType' error occurs
Ⅲ.I found that many issues are related to CUDA 0 and 1. I am using the GPU at CUDA:0, and there are errors reported in pytest. Will this affect my normal usage?

**Comment by RBirmiwal** [comment, marked as answer]
Hi @Wangzixu0426. Thank you for our interest in our library. We recently found some issues with our test_blocks.py (the errors you are encountering are the same ones we are also encountering). This will be fixed in an upcoming release shortly. 

However **the library, including all our blocks, is still operating just fine**; this bug lies in how we've constructed the test and has not scaled up to our newer methods. Thanks. If you have any more issues or questions feel free to let us know!
